# Calling functions from functions #

It is perfectly okay to call a function from within a function!

